verification/verifier.go

```diff
@@ -151,8 +151,12 @@ func (v *Verifier) processSignature(ctx context.Context, sigBlob []byte, sigMani
 
 	// check if we need to verify using a plugin
 	var pluginCapabilities []plugin.Capability
-	verificationPluginName := GetVerificationPlugin(outcome.SignerInfo)
-	if verificationPluginName != "" {
+	verificationPluginName, err := getVerificationPlugin(outcome.SignerInfo)
+	// use plugin, but getPluginName returns an error
+	if err != nil && err != errExtendedAttributeNotExist {
+		return err
+	}
```

Comment:
we should also check VerificationPluginMinVersion by calling the getVerificationPluginMinVersion func here.

Reply:
> we should also check VerificationPluginMinVersion by calling the getVerificationPluginMinVersion func here.

Updated. PTAL